src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -308,13 +308,20 @@ public function applyDefaults(ContainerBuilder $container, $serviceId, array $at
 
         $definition->addMethodCall('setLabel', [$label]);
 
+        $persistFilters = $container->getParameter('sonata.admin.configuration.filters.persist');
+        // override default configuration with admin config if set
         if (isset($attributes['persist_filters'])) {
-            $persistFilters = (bool) $attributes['persist_filters'];
-        } else {
-            $persistFilters = (bool) $container->getParameter('sonata.admin.configuration.filters.persist');
+            $persistFilters = $attributes['persist_filters'];
```

Comment:
Could be located here? @Soullivaneuh , before we had a cast to (bool) and now we don't. Maybe the value is getting there but not with the correct type?

Reply:
well "true" would also be considered as a true so not sure that is it

Reply:
"true" !== true

Reply:
if strict is used 😄

Reply:
https://github.com/sonata-project/SonataAdminBundle/blob/0344393db83244b7422e75984c99a1bf7eb2de13/src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php#L314-L323

and it is not